src/bundle/Editor.ts

```diff
@@ -313,6 +319,8 @@ export class EditorImpl extends SafeEventEmitter<EventMapInt> implements EditorI
 
         this.#renderStorage = opts.renderStorage;
         this.#fileUploadHandler = handlers.uploadFile;
+        this.#parseInsertedUrlAsImage =
+            wysiwygConfig.extensionOptions?.imgSize?.parseInsertedUrlAsImage;
```

Comment:
I suggest creating the same prop in [markup config](https://github.com/gravity-ui/markdown-editor/blob/main/src/bundle/types.ts#L68), so as not to take it from WYSIWYG config for for markup.